cloudbuild.yaml

```diff
@@ -25,8 +25,10 @@ tags:
 - ${_GIT_TAG}
 - ${_PULL_BASE_REF}
 
-images:
-  - 'gcr.io/$PROJECT_ID/cip:$_GIT_TAG'
-  - 'gcr.io/$PROJECT_ID/cip:latest'
-  - 'gcr.io/$PROJECT_ID/cip-auditor:$_GIT_TAG'
-  - 'gcr.io/$PROJECT_ID/cip-auditor:latest'
+# TODO: Temporarily disabling this check, which is currently failing in
```

Comment:
Once we remove bazel (which is doing the image building (and pushing) *without* tagging them locally, which breaks this stanza), we should be able to reinstate this.

Reply:
@listx -- Thanks for the context! I had a feeling it was a bazel oddity at work here.